chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ApiKeyController.java

```diff
@@ -82,18 +86,37 @@ private String create(final ApiKeyRegistration registration) throws EntityCreati
         return "{\"keyRegistered\" : \"" + apiKey + "\"}";
     }
 
+    @ApiOperation(value = "Sends an email validation to user requesting a new API key.",
+            notes = "Anyone wishing to access the methods listed in this API must have an API key. This request "
+                      + "will create an email invitation and send it to the supplied email address.  The "
+                      + "purpose of the invitation is to validate the email address of the potential API user.")
+    @RequestMapping(value = "/request", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = "application/json; charset=utf-8")
+    public String request(@RequestBody ApiKeyRegistration registration) throws ValidationException {
+        return "{\"success\" : \"" + apiKeyManager.createRequest(registration).toString() + "\"}";
```

Comment:
Same type of comment about writing out the JSON string ourselves. If that's an accepted practice then I'll live but I don't really like it. We do it in many places but I feel like that is all older CHPL code. Could we just key off the response code to know if it was successful rather than need this "success" key?